src/raftstore/store/msg.rs

```diff
@@ -67,9 +67,16 @@ pub enum Msg {
     SplitCheckResult {
         region_id: u64,
         epoch: RegionEpoch,
+        // It's a data key, starts with `DATA_PREFIX_KEY`.
```

Comment:
We may support meta key later.

Reply:
can be same type with SplitRegion.split_key